app/src/main/java/io/neurolab/utilities/DataLogger.java

```diff
@@ -0,0 +1,37 @@
+package io.neurolab.utilities;
+
+import android.os.Environment;
+
+import java.io.File;
+
+public class DataLogger {
```

Comment:
There a filepathutil you've created once. Why we creating another class for file handling? https://github.com/fossasia/neurolab-android/blob/development/app/src/main/java/io/neurolab/utilities/FilePathUtil.java

Reply:
I made it actually for more manipulating functions for LoggedData files (in future) to be kept here. Okay then will move this func to there for now.

Reply:
Move similar functions to similar classes.